client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -157,6 +161,18 @@ def _set_scene_path(self, current_file, use_published):
         self.scene_path = file_path
         self.log.info("Using {} for render/export.".format(file_path))
 
+    def _append_job_output(self, instance, job_info):
```

Comment:
First what came into my mind when I read the function name is that this appeds stdout.
```suggestion
    def _append_job_output_paths(self, instance, job_info):
```